core/templates/core/base_international.html

```diff
@@ -1,5 +1,9 @@
 {% load static from staticfiles %}
 {% load trans from i18n %}
+{% load build_email_link from social_tags %}
+{% load build_linkedin_link from social_tags %}
+{% load build_twitter_link from social_tags %}
+{% load build_facebook_link from social_tags %}
```

Comment:
this is ok. note in future you caan do `{% load build_email_link build_linkedin_link from social_tags %}`

Reply:
ah! didn't know that